clients/include/testing_dot_batched_ex.hpp

```diff
@@ -105,7 +105,7 @@ hipblasStatus_t testing_dot_batched_ex_template(const Arguments& argus)
     double gpu_time_used, hipblas_error_host, hipblas_error_device;
 
     // Initial Data on CPU
-    hipblas_init(hy, true, true);
+    hipblas_init(hy, true, false);
```

Comment:
If you aren't alternating signs won't we have potential large N unit check failures?

Reply:
hx has alternating input below this.